marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/writerScenario2.spec.tsx

```diff
@@ -59,7 +59,7 @@ describe("Entity Modeling: Writer Role", () => {
     structuredTypeModal.getAddButton().click();
     propertyModal.getYesRadio("multiple").click();
     propertyModal.getSubmitButton().click();
-    cy.contains("A property already exists with a name of Address");
+    cy.get("[data-testid=\"propery-name-error\"]").contains("A property or structured type are already using the name Address. A property cannot use the same name as an existing property or structured type.");
```

Comment:
Looks good @aniamy. Could we try to move the locating of the element into a helper function in 'property-modal.tsx'. This way we can re-use if necessary and just change the error being passed in.

For example, 
`cy.verifyPropertyNameError("A property or structured type...");`

The helper function should take in a string which is the error message and then use findByTestId("property-name-error").contains("example").